File: eSalon/eSalon.Model/Usluga.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace eSalon.Model
{
    public class Usluga
    {
        public int UslugaId { get; set; }

        public string Naziv { get; set; } = null!;

        public string Opis { get; set; } = null!;

        public decimal Cijena { get; set; }

        public int Trajanje { get; set; }

        public byte[]? Slika { get; set; }

        public DateTime DatumObjavljivanja { get; set; }

        public int VrstaId { get; set; }
        public string VrstaUslugeNaziv { get; set; }=string.Empty;
    }
}

File: eSalon/UI/esalon_mobile/lib/models/usluga.dart
import 'package:json_annotation/json_annotation.dart';

part 'usluga.g.dart';

@JsonSerializable()
class Usluga {
  int? uslugaId;
  String? naziv;
  String? opis;
  double? cijena;
  int? trajanje;
  String? slika;
  String? datumObjavljivanja;
  int? vrstaId;
  String? vrstaUslugeNaziv;

  Usluga({
    this.uslugaId,
    this.naziv,
    this.opis,
    this.cijena,
    this.trajanje,
    this.slika,
    this.datumObjavljivanja,
    this.vrstaId,
    this.vrstaUslugeNaziv,
  });

  factory Usluga.fromJson(Map<String, dynamic> json) => _$UslugaFromJson(json);
  Map<String, dynamic> toJson() => _$UslugaToJson(this);
}